### ISSUE TITLE
Would be useful to have a few more tracers of solute movement available

### ISSUE THREAD

**Issue by sno036** [APSIMInitiative/ApsimX#9358]
### Describe the new feature

Maybe just name them Tr1, Tr2 etc. Just additional solutes that SoilWater would consider to be mobile (and optional) and that for SWIM would get the usual parameters.

**Comment by hut104** [comment]
@sno036 Chloride test simulations are already within the test set, with initialisation via the chemical node in the soil.  See the Hudson simulations (tests/validation/system) as an example test set.

Is that what you are looking for here?

**Comment by sno036** [comment]
Not really. This is adding tracers (eventually multiple different ones) during a simulation. I've made a Manager that can use Cl as a tracer for one person but something more general would be useful.

**Comment by sno036** [comment]
@MWillgerodt - I have raised a pull request (#9394) to allow Cl to be added using Operations. Once that is merged and you update to that version you can add Cl from Operations using something like:

2005-01-15 [AddTracer].Script.AddTracer(Amount: 200, Depth: 1)

where "AddTracer" is the name of the script in the ManagementToolbox. It is a bit clunky in that you will have to either enter a date into the Manager UI that is outside your simulation window or set the amount to zero to effectively disable the Manager addition of Cl. 

Hope that makes sense. Email me if not,

**Comment by MWillgerodt** [comment]
Adding to this issue, it would be nice to have Tracers that behave exactly like fertilizer N or P or .... With this not only the water flow, but also the uptake of for example the fertilizer N by plants could be visualized. Like a marker isotope N, which isn't disciminated against.

**Comment by sno036** [comment]
I can see what you mean and can see the utility but this would be rather challenging I think.

**Comment by sno036** [comment]
@MWillgerodt - @hol353 has considerably cleaned up my code. This has changed the syntax for adding the tracer from Operations. It is now as below and should be in release tomorrow.

2005-01-15 [Cl].AddAtDepth(amount:100, depth: 1)

or if you want it every year then

15-Jan [Cl].AddAtDepth(amount:100, depth: 1)
